namespace-resources-cli-template/resources/prototype/templates/cd.yaml

```diff
@@ -7,7 +7,7 @@ on:
   workflow_dispatch:
   push:
     branches:
-      - 'branch-name'
+      - "branch-name"
```

Comment:
This should point towards the branch you want to use for cd, normally that's main.

Reply:
This is not something I've changed. It's the code formatters that run on this repo